src/frontend/frontend/templates/macros/forms.html

```
@@ -84,77 +84,42 @@
-               class="toggle toggle-primary peer {{ input_class }}"
-               {% if name %}aria-label="{{ name }}"{% endif %}
-               {% if required %}required{% endif %}
-               {% if value == yes_val %}checked{% endif %}
-               {% if disabled %}disabled{% endif %} />
-        <span class="text-base-content peer-checked:hidden">{{ no_val }}</span>
-        <span class="text-base-content hidden peer-checked:inline">{{ yes_val }}</span>
-      {% endmacro %}
+{% macro form_switch_content(name, value, yes_val='Yes', no_val='No', required=True, preserve=True, input_class='', disabled=False, input_id=None) %}
+  <!-- Switch with {{ value }} -->
+  <input type="hidden" name="{{ name }}" value="{{ no_val }}" {% if disabled %}disabled{% endif %} />
+  <input id="{{ input_id if input_id else name }}"
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)